server.go

```diff
@@ -46,16 +46,16 @@ type ServerRelations struct {
 
 //ServerCreateRequest JSON struct of a request for creating a server
 type ServerCreateRequest struct {
-	Name            string                        `json:"name"`
-	Memory          int                           `json:"memory"`
-	Cores           int                           `json:"cores"`
-	LocationUUID    string                        `json:"location_uuid"`
-	HardwareProfile string                        `json:"hardware_profile,omitempty"`
-	AvailablityZone string                        `json:"availability_zone,omitempty"`
-	Labels          []string                      `json:"labels,omitempty"`
-	Relations       *ServerCreateRequestRelations `json:"relations,omitempty"`
-	Status          string                        `json:"status,omitempty"`
-	AutoRecovery    bool                          `json:"auto_recovery,omitempty"`
+	Name            string   `json:"name"`
+	Memory          int      `json:"memory"`
+	Cores           int      `json:"cores"`
+	LocationUUID    string   `json:"location_uuid"`
+	HardwareProfile string   `json:"hardware_profile,omitempty"`
+	AvailablityZone string   `json:"availability_zone,omitempty"`
+	Labels          []string `json:"labels,omitempty"`
+	Status          string   `json:"status,omitempty"`
+	AutoRecovery    *bool    `json:"auto_recovery,omitempty"`
+	//Relations       *ServerCreateRequestRelations `json:"relations,omitempty"`
```

Comment:
why is it commented?

Reply:
It disappears from the  official API docs and I have tested it. The server returns "Bad request" when we include "relations" field.

Reply:
Just in case it will be back in the future, I just commented this line of code.

Reply:
Then, I would prefer removing all `Relations` and `ServerCreateRequestRelations`.

Reply:
I don't know if it will be back in the future. Is it necessary to remove these lines of code?

Reply:
I don't like keeping codes that are unused. We can add this feature again when API supports it. That why we use git ` version-control system`.

Reply:
Alright then. I will update the code in my branch, I think you will need to pull the update.